birdhouse/components/cowbird/docker-compose-extra.yml

```diff
@@ -0,0 +1,82 @@
+version: "3.4"
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "10"
+
+services:
+  cowbird:
+    image: pavics/cowbird:${COWBIRD_VERSION}-webservice
+    container_name: cowbird-web
```

Comment:
name the container_name to match the service name ... easier for the users follow